x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -10999,7 +10999,6 @@
     "xpack.monitoring.alerts.licenseExpiration.actionGroups.default": "默认值",
     "xpack.monitoring.alerts.licenseExpiration.newSubject": "新 X-Pack Monitoring：许可证到期",
     "xpack.monitoring.alerts.licenseExpiration.resolvedSubject": "已解决 X-Pack Monitoring：许可证到期",
-    "xpack.monitoring.alerts.licenseExpiration.ui.firingMessage": "此集群的许可证将在 #relative 后，即 #absolute到期",
```

Comment:
this looks wrong? merge conflict resolved wrong?

Reply:
There was an [i18n failure on the previous build](https://github.com/elastic/kibana/runs/565761033) when merging with master so I checked out locally, ran `node scripts/i18n_check.js --fix`, and these are the changes from that.